### ISSUE TITLE
Taiga launches wrong player

### ISSUE THREAD

**Issue by Nekuroi** [erengy/taiga#656]
I'm using wmp x64 and taiga is launching the x86 version when playing from list.
File associations are fine since they launch the correct version in windows.

It would be nice to have an option to choose the desired player.

Using W10 Pro 1803

**Comment by ShaunV2** [comment]
> I''d assume that Windows launches the 32-bit version of WMP because Taiga itself is a 32-bit application.

This is false, if you have the video itself setup correctly (go to the properties of the video and make sure it is opened with the 64-bit version) to use the 64-bit version it doesn't matter at all if Taiga is 32-bit as I just tested, any videos I have Taiga run will use MPC-HC 64-bit.
![screenshot 61](https://user-images.githubusercontent.com/14097860/42939988-a91003da-8b0c-11e8-9511-730523534c28.png)
